x/ethbridge/keeper/blacklist.go

```diff
@@ -59,9 +59,11 @@ func (k Keeper) SetBlacklist(ctx sdk.Context, msg *types.MsgSetBlacklist) error
 
 func (k Keeper) GetBlacklist(ctx sdk.Context) []string {
 	var addresses []string
+
 	store := ctx.KVStore(k.storeKey)
-	iter := store.Iterator(types.BlacklistPrefix, nil)
+	iter := sdk.KVStorePrefixIterator(store, types.BlacklistPrefix)
```

Comment:
Does store.Iterator() need to be replaced throughout? I believe it also appears in the oracle keepr

Reply:
Good point, I updated it in the oracle keeper as well, and extended the relevant unit test.
It doesn't appear anywhere else.